pyproject.toml

```diff
@@ -1,48 +1,97 @@
 [build-system]
+requires = ["setuptools>=45",
+            "setuptools_scm[toml]>=6.2",
+]
+build-backend = 'setuptools.build_meta'
 
-requires = ["setuptools",
-            "setuptools_scm",
-            "wheel"]
+[project]
+name = "hermes_core"
+description = "A Python package to support the HERMES instrument packages."
+readme = "README.rst"
+dynamic = ["version"]
+authors = [{name = "Steven Christe", email="steven.d.christe@nasa.gov"}]
+license = {file = "LICENSE.rst"}
+requires-python = ">=3.7"
+keywords = ["hermes", "nasa mission", "space weather"]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: Apache 2.0",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+dependencies = [
+  'astropy>=4.1.0',
+  'numpy>=1.16.0',
+  'sunpy>=3.1.4',
+]
 
-build-backend = 'setuptools.build_meta'
+[project.optional-dependencies]
+dev = [
+  'coverage>=5.0.3',
+  'pytest',
+  'pytest-astropy',
+  'pytest-cov',
+  'black',
+  'flake8',
+  'coverage[toml]'
+]
+
+docs = [
+  'sphinx',
+  'sphinx-automodapi'
+]
+
+test = [
+  'pytest',
+  'pytest-astropy',
+  'pytest-cov',
+  'coverage[toml]'
+]
+
+style = [
+  'black',
+  'flake8'
+]
+
+[tool.setuptools.package-data]
+hermes_core = ["data/*"]
+
+[tool.setuptools]
+packages = ["hermes_core"]
+
+[tool.setuptools_scm]
+write_to = "hermes_core/_version.py"
```

Comment:
I verified this works by copying the package locally from this PR branch and running the `python -m setuptools_scm` command:
![setuptool_scm](https://user-images.githubusercontent.com/19628176/206760372-ee9ecec7-826f-4c89-862c-8a4711ee5da6.png)